src/test/smart-components/portfolio/portfolio-item-detail/__snapshots__/item-detail-info-bar.test.js.snap

```diff
@@ -216,7 +216,7 @@ exports[`<ItemDetailInfoBar /> should render correctly 1`] = `
                     zIndex={9999}
                   >
                     <span>
-                      11 months ago
+                      12 months ago
```

Comment:
@Hyperkid123 - the test that checks this snapshot will pass now, but fail next month. I would either remove the test or remove the dates and regenerate the snapshot without the dates

Reply:
I will add a mock to the test that will return always one year old date. That should be enough.